src/docs/database-and-api-management/crunchydb-workshop.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Workshop - Learn to use Postgres on OpenShift with CrunchyDB
```

Comment:
I know the name has changed already - but what are your thoughts on calling this a 'lab' rather than a workshop- since it is a set of exercises. This would be consistent with our usage throughout the OpenShift 101 labs as compared with the 101 workshops.

Reply:
Hmm. I'd be open to changing it. I've been using the word "workshop" mostly because I feel like a lab is associated with a lecture, whereas a workshop is standalone. But I'm not sure if that's a consistent definition that everyone else uses, or if that's just me, lol.

Reply:
Yeah it's funny, I feel the same way about 'workshop' having more of a live connotation. It's probably no big deal either way, the main thing is that this new learning option will be available. Looking forward to giving it a try.

Reply:
Hi @caggles and @mtspn thank you for your feedback. Can you let me know your final thoughts on this?

 So this is what I found: 

A workshop and a lab, though similar, often serve slightly different purposes in a learning or training context:

**Workshop**:

- Generally involves interactive, instructor-led sessions
- Participants often work on practical exercises or projects during the session
- Focuses on teaching new skills or concepts in a collaborative environment
- May include discussions, Q&A sessions, and hands-on activities guided by the instructor

**Lab**:

- Typically more self-directed and hands-on
- Participants work through a series of exercises or experiments, often at their own pace
- Emphasizes practical application and experimentation
- Often used in academic or technical training settings to reinforce theoretical knowledge through practice

Also some _other_ suggestions based on structured learning with multimedia (YouTube): 

 - Training module: Learn to use Postgres on OpenShift with CrunchyDB
-  Tutorial series: Learn to use Postgres on OpenShift with CrunchyDB
-  Comprehensive guide: Learn to use Postgres on OpenShift with CrunchyDB
-  Interactive training: Learn to use Postgres on OpenShift with CrunchyDB
-  Video guide: Learn to use Postgres on OpenShift with CrunchyDB

Reply:
'Lab' or 'Self-directed lab' would be how I'd describe this. Happy to go with what you think is best @Pilargit12

Reply:
Through Rocket.chat - Cailey mentioned Tutorial sounded good!

Reply:
> 'Lab' or 'Self-directed lab' would be how I'd describe this. Happy to go with what you think is best @Pilargit12

Oh sorry Matt, only read this through after!